### PR TITLE
Edit pass on Abstract, Introduction, and General Setting

### PR DESCRIPTION
**Pull request by bifurcation**
Mainly focused on cleaning up grammar, but also fixing some things that have gone stale, like the idea that the MLS protocol is "abstract".

### CODE REVIEW

File: draft-ietf-mls-architecture.md [hunk 1]
```diff
@@ -296,13 +251,13 @@ public values such as a name (an identity), a public encryption key
...
+to other clients or members in the Group.
 In some messaging systems, clients belonging to the same user must
-all share the same identity key pair, but MLS does not assume this.
+all share the same signature key pair, but MLS does not assume this.
 
-Users will typically own multiple clients, potentially one or more per
-end-user devices (phones, web clients or other devices...) and may
+Users will often use multiple clients, potentially one or more per
+device (phones, web clients or other devices...), and may
 choose to authenticate using the same signature key across devices,
```
Comment (kkohbrok) [inline draft-ietf-mls-architecture.md:261]: While you're at it, I believe it's not really possible anymore to use the same signature key across client. At least not if two client are meant to be members of the same group. The application could choose to authenticate the credentials based on some other key material, but that really depends on how the AS is designed. I would suggest we discuss the different options in the section describing the AS instead.

**Review by kkohbrok — COMMENTED** [review]
Looks good! Just two comments on client identity.

File: draft-ietf-mls-architecture.md [hunk 2]
```diff
@@ -296,13 +251,13 @@ public values such as a name (an identity), a public encryption key
 and a public signature key. Ownership of a client by a user is
 determined by the fact that the user has knowledge of the
 associated secret values. When a client is part of a Group, it is
-called a Member and its signature key pair uniquely defines its
-identity to other clients or members in the Group.
+called a Member and its signature key pair uniquely identifies it
+to other clients or members in the Group.
```
[the remark this answers is not in the document]
Comment (bifurcation) [reply draft-ietf-mls-architecture.md:255]: Yeah, but I don't think we want to get into that detail here.  The fault here is probably in focusing on identity as opposed to saying more directly that the key pair is used to authenticate messages from that client.  I'll try to reword.

File: draft-ietf-mls-architecture.md (hunk 1 above)
Comment (bifurcation) [reply draft-ietf-mls-architecture.md:261]: Fair point, will try to reword.

**Comment by beurdouche** [comment]
This looks good ! Thank you !

**Pull request merged**